rts/motoko-rts/src/blob_iter.rs

```diff
@@ -21,7 +22,7 @@ unsafe fn blob_iter<M: crate::memory::Memory>(mem: &mut M, blob: Value) -> Value
     (*iter_array).len = 2;
 
     iter_array.initialize(ITER_BLOB_IDX, blob, mem);
-    iter_array.set_scalar(ITER_POS_IDX, Value::from_scalar(0));
+    iter_array.set(ITER_POS_IDX, Value::from_scalar(0), mem);
```

Comment:
Looks I forgot to tag these scalars, though probably ok since just internal state of the iterator. Might cause a problem with graph serialization if some stable data structure store an iterator in a field obscure by subtyping.

Reply:
https://github.com/dfinity/motoko/issues/4466 tracks this

Reply:
Indeed, this could be an issue. Thanks a lot for noticing.

Reply:
I believe we have tested this case with https://github.com/dfinity/motoko/pull/4562. Iterators seem to be hidden behind closures and thus not analyzed by graph copy.